include/picongpu/fields/background/templates/twtsfast/BField.hpp

```diff
@@ -0,0 +1,182 @@
+                 * The other origin coordinates (x and z) default to globally centered values
+                 * with respect to the simulation volume.
+                 */
+                PMACC_ALIGN(focus_y_SI, float_64 const);
+                /* Laser wavelength [meter] */
+                PMACC_ALIGN(wavelength_SI, float_64 const);
+                /* TWTS laser pulse duration [second] */
+                PMACC_ALIGN(pulselength_SI, float_64 const);
+                /* line focus height of TWTS pulse [meter] */
+                PMACC_ALIGN(w_x_SI, float_64 const);
+                /* interaction angle between TWTS laser propagation vector and the y-axis [rad] */
+                PMACC_ALIGN(phi, float_X const);
+                /* Takes value 1.0 for phi > 0 and -1.0 for phi < 0. */
+                PMACC_ALIGN(phiPositive, float_X);
+                /* propagation speed of TWTS laser overlap
+                   normalized to the speed of light. [Default: beta0 = 1.0] */
+                PMACC_ALIGN(beta_0, float_X const);
+                /* If auto_tdelay=FALSE, then a user defined delay is used. [second] */
+                PMACC_ALIGN(tdelay_user_SI, float_64 const);
+                /* Make time step constant accessible to device. */
+                PMACC_ALIGN(dt, float_64 const);
+                /* Make length normalization constant accessible to device. */
+                PMACC_ALIGN(unit_length, float_64 const);
+                /* TWTS laser time delay */
+                PMACC_ALIGN(tdelay, float_64);
+                /* Should the TWTS laser time delay be chosen automatically, such that
+                 * the laser gradually enters the simulation volume? [Default: TRUE]
+                 */
+                PMACC_ALIGN(auto_tdelay, bool const);
+                /* Polarization of TWTS laser */
+                PMACC_ALIGN(pol, PolarizationType const);
+
+                /** Magnetic field of the TWTS laser
+                 *
+                 * \param focus_y_SI the distance to the laser focus in y-direction [m]
+                 * \param wavelength_SI central wavelength [m]
+                 * \param pulselength_SI sigma of std. gauss for intensity (E^2),
+                 *  pulselength_SI = FWHM_of_Intensity / 2.35482 [seconds (sigma)]
+                 * \param w_x beam waist: distance from the axis where the pulse electric field
+                 *  decreases to its 1/e^2-th part at the focus position of the laser [m]
+                 * \param phi interaction angle between TWTS laser propagation vector and
+                 *  the y-axis [rad, default = 90.*(PI/180.)]
+                 * \param beta_0 propagation speed of overlap normalized to
+                 *  the speed of light [c, default = 1.0]
+                 * \param tdelay_user manual time delay if auto_tdelay is false
+                 * \param auto_tdelay calculate the time delay such that the TWTS pulse is not
```

Comment:
please use `@param` for doxygen comments

Reply:
I applied this change also to the keywords `@tparam` and `@return`.